src/main/cpp/mechanisms/DragonTale/DragonTale.cpp

```diff
@@ -1049,6 +1049,7 @@ void DragonTale::Update()
 	{
 		m_AlgaeTalonFXS->SetControl(*m_AlgaeTalonFXSActiveTarget);
 	}
+	RobotState::GetInstance()->PublishStateChange(RobotStateChanges::StateChange::ElevatorHeight_Int, static_cast<double>(GetElevatorHeight()));
```

Comment:
Needs to publish the elevator height as a inch, this method is overloaded as many different types.